lib/countly.js

```diff
@@ -1701,15 +1724,15 @@
         } catch (e) {
             // fallback
             log("Failed XML HTTP request", e);
-            if (typeof callback === 'function') { callback(true, params); }
+            if (typeof callback === 'function') { callback(true, params, this.responseText); }
```

Comment:
I think here this belongs to other context. But we don't need it actually, as request failed, we dont have any response